SPECS/ocaml-ppx-here/ocaml-ppx-here.spec

```diff
@@ -0,0 +1,166 @@
+# OCaml packages not built on i686 since OCaml 5 / Fedora 39.
+ExcludeArch: %{ix86}
+
+Name:           ocaml-ppx-here
+Version:        0.16.0
+Release:        9%{?dist}
+Summary:        Expands [@here] into its location
```

Comment:
I think we're missing the `Vendor` and `Distribution` tags. Please double-check other specs as well.

Reply:
Added tags